src/splat.rs

```diff
@@ -492,6 +504,49 @@ pub(crate) fn splat(
         })
         .collect_into_vec(&mut results);
 
+    match item.payload.kind {
+        PayloadKind::SdkLibs => {
+            // Symlink sdk/lib/{sdkversion} -> sdk/lib, regardless of filesystem case sensitivity.
+            let sdk_version = find_sdk_version(&item.payload.filename);
+            let mut versioned_linkname = roots.sdk.clone();
+            versioned_linkname.push("lib");
+            versioned_linkname.push(sdk_version);
+            symlink(".", &versioned_linkname)?;
```

Comment:
Just tried this in xwin 0.2.2, looks like this will fail when more than one architectures are specified?

```
Error: failed to splat Win10SDK_10.0.20348_libs_aarch64.msi

Caused by:
    0: unable to symlink from /home/runner/.cache/cargo-xwin/xwin/sdk/lib/10.0.20348 to .
    1: File exists (os error 17)
```

https://github.com/messense/cargo-xwin/runs/6448797813?check_suite_focus=true

Reply:
Whoopsie! Thank you for fixing that 😅